.gitignore

```diff
@@ -29,3 +29,4 @@ __pycache__
 /tests/.vscode
 /sonobuoy-output
 *.tmp
+k3s.exe
```

Comment:
probably unnecessary, build into /bin or something?

Reply:
This has been fixed.